interface/modules/custom_modules/oe-module-comlink-telehealth/tests/Tests/Unit/TeleHealthUserRepositoryTest.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+/**
+ * Handles the TeleHealthUserRepository Unit Tests
+ *
+ * @package openemr
+ * @link      http://www.open-emr.org
+ * @author    Stephen Nielson <snielson@discoverandchange.com>
+ * @copyright Copyright (c) 2022 Comlink Inc <https://comlinkinc.com/>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ */
+
+namespace Comlink\OpenEMR\Modules\TeleHealthModule;
+
+use Comlink\OpenEMR\Modules\TeleHealthModule\Models\TeleHealthUser;
+use Comlink\OpenEMR\Modules\TeleHealthModule\Repository\TeleHealthUserRepository;
+use OpenEMR\Common\Database\QueryUtils;
+use PHPUnit\Framework\TestCase;
+
+class TeleHealthUserRepositoryTest extends TestCase
+{
+    const TEST_USERNAME = "phpunit-test-username";
+    const TEST_PASSWORD = "randomToken";
+
+    protected function tearDown(): void
+    {
+        parent::tearDown(); // TODO: Change the autogenerated stub
+        QueryUtils::sqlStatementThrowException("DELETE FROM " . TeleHealthUserRepository::TABLE_NAME
+            . " WHERE username LIKE '%" . self::TEST_USERNAME . "%'", []);
```

Comment:
rec using binding for `% . self::TEST_USERNAME . %` just to be safe since never know what devs may do in future and then know it will never turn into a sql injection vulnerability in future

Reply:
Fixed the test case to use binding.